expression/builtin_time.go

```diff
@@ -638,15 +638,16 @@ func builtinDateArith(args []types.Datum, ctx context.Context) (d types.Datum, e
 	// Op is used for distinguishing date_add and date_sub.
 	// args[0] -> Op
```

Comment:
Hi @hanfei1991 , I think such a refactoring is not strongly related to the topic of this PR. Can I do it in the next PR?